src/cacheDisplay.html

```diff
@@ -178,6 +178,12 @@
                     </tbody>
                 </table>
             </md-table-container>
+            <style>
```

Comment:
Could this be in a css file?

Reply:
Good point. I just moved it to an external stylesheet.